recipes/opensciencereasoning/configs/SDG_pipeline/gpt-oss-seed-data_with_gt.yaml

```diff
@@ -0,0 +1,159 @@
+cluster: local
+base_output_dir: "/workspace/opensciencereasoning/gpt-oss"
+expname: "gpt-oss-generate-data"
+suffix: ""  # Suffix for experiment names
+dataset_name: ""  # Optional dataset name for id generation
+
+# Input file for the first stage (generate_solutions)
+# It must include the field "problem"
+# All the data in this file will be processed with the same prompt
+# Make sure the problems fit the prompt you are using
+input_file: ""
+
+# Can define initial dependency for the `generate_solutions` stage to run after
+initial_dependency: None
+
+# Define the full sequence of stages for this mode
+pipeline_stages:
+  - filter_problems          # Filter problems based on format
+  - decontaminate            # Decontaminate problems
+  - topics_labeling          # Label topics and subtopics
+  - difficulty_estimation    # Estimate difficulty of problems
+  - aggregate                # Aggregate all the metadata into a single file
+  - filter_solutions         # Filter solutions
+
+# Directory structure configuration
+directories:
+  step-0-filter-problems: ${base_output_dir}/solution-sdg/step-0-filter-problems
+  step-1-decontaminate: ${base_output_dir}/solution-sdg/step-1-decontaminate
+  step-2-topics-labeling: ${base_output_dir}/solution-sdg/step-2-topics-labeling
+  step-3-difficulty-estimation: ${base_output_dir}/solution-sdg/step-3-difficulty-estimation
+  step-4-aggregate: ${base_output_dir}/solution-sdg/step-4-aggregate
+  step-5-filter-solutions: ${base_output_dir}/solution-sdg/step-5-filter-solutions
+
+# Stage-specific configurations
+stages:
+  filter_problems:
+    output_dir: ${directories.step-0-filter-problems}
+    input_file: ${input_file}
+    dataset_name: ${dataset_name}
+    problem_field: problem
+    expected_answer_field: expected_answer
+    id_field: id
+    remove_expected_answer: False
+    remove_images: True
+    num_options: null
+    option_format_regex: null
+    deduplicate: True
+    dependencies:
+      - initial_dependency
+
+  decontaminate:
+    output_dir: ${directories.step-1-decontaminate}
+    input_file: ${directories.step-0-filter-problems}/final_result.jsonl
+    test_sets:
+      - ["hle", "text"]
+      - ["mmlu", "test"]
+      - ["mmlu-pro", "test"]
+      - ["gpqa", "diamond"]
+    model: /hf_models/Qwen2.5-32B-Instruct
```

Comment:
consider using `Qwen/Qwen2.5-32B-Instruct` here and everywhere else to avoid extra step of manually downloading the model